.env.example

```
@@ -28,4 +28,12 @@ REDDIT_CLIENT_SECRET = <REDDIT_CLIENT_SECRET>
 
 CLOUDFLARE_API_TOKEN = <CLOUDFLARE_API_TOKEN>
 CLOUDFLARE_KV_NAMESPACE_ID = <CLOUDFLARE_KV_NAMESPACE_ID>
-CLOUDFLARE_ACCOUNT_ID = <CLOUDFLARE_ACCOUNT_ID>
\ No newline at end of file
+CLOUDFLARE_ACCOUNT_ID = <CLOUDFLARE_ACCOUNT_ID>
```

dotenv-linter (3.3.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)